scripts/setup.sh

```diff
@@ -6,17 +6,28 @@
 #
 # Commands that may fail have "|| return" to avoid continuing or interfering with terminal.
 
-if [ ! -z $NVM_DIR ]; then # skip nvm steps if not available
-  echo 'Configuring node 12...'
-  source ~/.nvm/nvm.sh
-  nvm install || return
+# Install yarn if it does not exist.
+if ! yarn versions &> /dev/null; then
```

Comment:
What do you think of using this for checking if a tool exists? I like the conciseness and the fact that its not actually calling it. @pvinis shared this approach in [this slack thread](https://artsy.slack.com/archives/CP9P4KR35/p1626998393239400?thread_ts=1626988667.231100&cid=CP9P4KR35)
```suggestion
if ! which yarn > /dev/null; then
```

Reply:
Much better 👌